src/js/modules/core/components/PageFooter.js

```diff
@@ -72,7 +87,11 @@ const PageFooter = ({classes, sectionsWithSubsections}) => {
   return (
```

Comment:
you aren't using MUI in all its glory. i like that you figured out flexbox, but definitely use `<Col>`'s also. Moreover, in the design, I've changed column and gutter widths. Which is why declaring `width` in the JSS is a terrible idea. Instead, declare the widths of the `<Col>` and determine gutters by padding between `<Col>`s.